ghcup

```diff
@@ -459,14 +459,14 @@ edo()
 # Wrapper around 'make', may call 'gmake' if it exists.
 emake() { # avoid re-checking for gmake
     if [ -n "${MAKE}" ] ; then
-        edo ${MAKE} "$@"
+        edo "${MAKE}" "$@"
```

Comment:
don't believe everything shellcheck tells you. Try the following in your shell:

```
$ MAKE="make -v"
$ ${MAKE}
GNU Make 4.2.1
Built for x86_64-pc-linux-gnu
Copyright (C) 1988-2016 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.
$ "${MAKE}"
bash: make -v: command not found
```

No quote here.

Instead, disable the shellcheck  https://github.com/koalaman/shellcheck/wiki/Directive#disable